server.js

```diff
@@ -1,8 +1,19 @@
+require("dotenv").config();
```

Comment:
Our Prettier config uses single quotes rather than double quotes. Can you update with the latest master since it looks like this branch doesn't have Vicky's latest changes which has `.prettierrc` in the root directory?

Reply:
Done! :)